### DISCUSSION TITLE
file name length + folder depth

### DISCUSSION THREAD

**Discussion by Danny-dK** [UtrechtUniversity/yoda#168] [Q&A] [answered]
Are there any limitations in file name length or folder structure depth within Yoda?

**Comment by lwesterhof** [comment, marked as answer]
In iRODS zitten de volgende beperkingen op collection name en data object name:
```
coll_name       | character varying(2700) | not null
data_name       | character varying(1000) | not null
```
2700 karakters op de naam van de collectie en 1000 karakters voor de naam van het bestand

Wanneer je iRODS benaderd via bijvoorbeeld webDAV zit je met de beperkingen van je besturingssysteem, bijvoorbeeld: https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry
